### PR TITLE
[EDU-3181] feat: add add records

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page on How to add records.

[EDU-3181]: https://aziontech.atlassian.net/browse/EDU-3181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/add-records.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+menu_namespace: secureMenu
+---
+
+After defining your [intelligent DNS main settings](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/), you can add records to specify which IPs are associated with the domain and how **Intelligent DNS** should handle requests for the domain.
+
+---
+
+## Via RTM
+
+1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > select **Intelligent DNS**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/add-records.mdx:16]:
```suggestion
1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > **Intelligent DNS**.
```

File: src/content/docs/en/pages/secure-journey/add-records.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+2. Choose the intelligent DNS in which you want to add records from the list.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/add-records.mdx:17]:
```suggestion
2. From the list, choose the intelligent DNS in which you want to add records.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/secure-journey/add-records.mdx:17]: ficou meio confuso

File: src/content/docs/en/pages/secure-journey/add-records.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+3. Select the **Records** tab.
+4. Click the **Add Record** button.
+5. In **Name**, provide the new record's name. Depending on the [type of record](/en/documentation/products/intelligent-dns/#type) you want to use, there may exist specific formats and recommendations. For example: for a CNAME record, you can use the name `www`.
+6. In **Type**, select the type of record you want to add. Accepts the types `A`, `AAAA`, `ANAME`, `CAA`, `CNAME`, `DS`, `MX`, `NS`, `PTR`, `SRV`, and `TXT`.
+7. In **Value**, add the items for the DNS response to the registered record. The accepted values vary according to the [chosen type of record](/en/documentation/products/intelligent-dns/#type).
+8. In **TTl (seconds)**, choose the time, in seconds, a response can be cached for on a resolver server. Maximum value: ``.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/add-records.mdx:23]: conferir o value

File: src/content/docs/en/pages/secure-journey/add-records.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+9. In **Policy**, select between **Simple** or **Weighted**.
+10. If you select **Weighted** in **Policy**:
+    -  In **Weight**, specify the weight for each record. Accepts values from `0` to `255`.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/add-records.mdx:26]:
```suggestion
    - In **Weight**, specify the weight for each record. Accepts values from `0` to `255`.
```

File: src/content/docs/en/pages/secure-journey/add-records.mdx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    - In **Description** (optional), you can describe, for example, the distinction between records with same Name and Type. Accepts up to 45 characters.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/add-records.mdx:27]:
```suggestion
    - In **Description** (optional), you can describe, for example, the distinction between records with the same **Name** and **Type**. Accepts up to *45 characters*. 
```

File: src/content/docs/en/pages/secure-journey/add-records.mdx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    "policy": "simple",
+    "entry": "www",
+    "id": 12345,
+    "description": "description"
+  },
+  "schema_version": 3
+}
+```
+
+Wait a few minutes for the changes to propagate and your recorded will be created in the hosted zone you chose.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/add-records.mdx:81]:
```suggestion
Wait a few minutes for the changes to propagate and your records will be created in the hosted zone you chose.
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**